README.md

```diff
@@ -53,17 +181,219 @@ const createMyFirstNFT = async () => {
+
+```javascript
+const create createNFTManually = async () => {
+  try {
+    ...
+
+    // STEP 3.3 : It get the same parameters as the submitTxBlocking
+    // Here we submit the transaction hex value signed before. 
+    // We could have used the unsigned tx Hash and pass a keyring as a third parameter instead.
+    // Once again, we use here the tx hex signTxHash, from the previous step 2.
+    const submitTxHash = await submitTxNonBlocking(signTxHash, WaitUntil.BlockInclusion)
+
+    // Do something with the events recieved. 
+    console.log(submitTxHash);
+    ...
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+**Ternoa Token Standardization** : Awaiting finalised TRC versions (Ternoa's Request for Comment)
+
+## API Architecture : 
+
+Ternoa SDK handles the main features proposed by the Ternoa chain. It allows you to run every transaction from the chain pallet, make a query like asking for some constant's storage. We also provide you with a bunch of helpers and utility functions to help you enjoy the full experience.
+
+The main handlers are the ones below :
++ blockchain: Handles the CORE blockchain function.and the API brain randomely: initializes the API, execute transactions, query datas, batch transactions and much more.
++ account: the functions that allows you to generate a new seed and a keyring
++ balance: the Balance pallet with its extrinsics, query and storage.
++ nft: the NFT pallet with its extrinsics, query and storage.
++ marketplace: the Marketplace pallet with its extrinsics, query and storage.
++ events: the events list returned when `submitTxBlocking` function is triggered
+
+### Handlers architecture
+For those who are familiar with Polkadot, you will quickly recognize the design structure of the features. If you aren't aware, no worries, the basic principles are easy enough to grasp. Depending on the pallet or handler category, you'll retrive :
+
++ **Constants** to request the chain runtime constants.
++ **Storage** to query the chain state.
++ **Extrinsics** to execute the transactions.
++ _Utilities_ when available provides some additional usefull functions you can directly import in your project.
+
+#### Response format
+As it makes sense for us to provide you the easiest tools to build on the Ternoa chain, we also try to simplify the the response format of our functions as much as we can. Depending on if you want to get things done or if you go with the full customizable way and handle it yourself, we invite you to choose the right function : Events and features data will be provided directly on some function while only transaction hash hex will be returned on others.
```

Comment:
Typo error: 'to simplify the the response' => 'to simplify the response'

Reply:
Done